### PR TITLE
[BEAM-124] Flink Running WordCountIT Example

### PR DESCRIPTION
**Pull request by markflyhigh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

Using FlinkRunner and TestFlinkRunner to run WordCountIT example with flowing commands:

`mvn clean verify -pl examples/java -am -rf :beam-examples-java -DskipITs=false -DintegrationTestPipelineOptions='[ "--tempRoot=gs://clouddfe-testing-temp-storage", "--runner=org.apache.beam.runners.flink.FlinkRunner" ]'
`

and

`
mvn clean verify -pl examples/java -am -rf :beam-examples-java -DskipITs=false -DintegrationTestPipelineOptions='[ "--tempRoot=gs://clouddfe-testing-temp-storage", "--runner=org.apache.beam.runners.flink.TestFlinkRunner" ]'
`

### CODE REVIEW

**Comment by markflyhigh** [comment]
+R: @dhalperi

File: examples/java/pom.xml [hunk 1]
```diff
@@ -273,7 +273,6 @@
       <artifactId>beam-runners-direct-java</artifactId>
       <version>${project.version}</version>
       <scope>runtime</scope>
-      <optional>true</optional>
```
Comment (dhalperi) [inline examples/java/pom.xml]: Why is this change necessary for Flunk to run?

Comment (markflyhigh) [inline examples/java/pom.xml]: It's a cleanup. Since we remove dependency on beam-examples-java in other module, no longer need to set optional.

Comment (dhalperi) [inline examples/java/pom.xml]: Please remove from this PR if it is not necessary for the PR. Bundling cleanup and functional changes is generally not a good idea -- it complicates review, testing, identification of issues, and rollback of bad changes.

Comment (markflyhigh) [inline examples/java/pom.xml]: see. fix it.

**Comment by dhalperi** [comment]
LGTM
